src/Surfnet/ServiceProviderDashboard/Application/CommandHandler/Entity/RequestDeletePublishedEntityCommandHandler.php

```diff
@@ -62,9 +71,13 @@ public function handle(RequestDeletePublishedEntityCommand $command)
         );
         $entity = $this->queryClient->findByManageId($command->getManageId());
         $ticket = Ticket::fromManageResponse($entity, $command->getApplicant());
-        $issue = $this->ticketService->createIssueFrom($ticket);
-
-        $this->logger->info(sprintf('Created Jira issue with key: %s', $issue->key));
-        $this->ticketService->storeTicket($issue->key, $command->getManageId());
+        try {
+            $issue = $this->ticketService->createIssueFrom($ticket);
+            $this->logger->info(sprintf('Created Jira issue with key: %s', $issue->key));
```

Comment:
Don't you want to put the logging above the try-catch-block or at least at the top of the block?

Reply:
No, this log entry reports on successfully adding the Jira ticket. Awaiting a successful Jira response seems pivotal in logging this information.